remove_starfield/starfield.py

```diff
@@ -262,9 +263,15 @@ def subtract_from_image(self,
         subtracted : `SubtractedImage`
             A container class storing the subtracted image and all inputs
         """
-        image_holder = processor.load_image(file)
+        if isinstance(file, str):
+            image_holder = processor.load_image(file)
+        elif isinstance(file, ImageHolder):
```

Comment:
This looks like it'll reject NDCubes that you pass in. Are you planning to wrap those with an ImageHolder subclass, or should this be `elif hasattr(file, wcs) and hasattr(file, data)`?

If the latter, maybe the docstrings should be something like `file : str or ImageHolderLike`?